ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.js

```diff
@@ -23,17 +23,17 @@ const CustomElement = styled.div`
 const Visualization = styled.div`
   cursor: pointer;
   opacity: ${props => props.theme.minimumOpacity}
+  padding: ${props => props.theme.largeSpacing}
   :hover {
     opacity: 1
   }
 `
 const VisualizationChoice = styled.div`
```

Comment:
❗️ VisualizationChoice appears to be unused, so these styles should move up into Visualization.
```suggestion
const Visualization = styled.div`
  cursor: pointer;
  opacity: ${props => props.theme.minimumOpacity};
  padding: ${props => props.theme.largeSpacing};
  :hover {
    opacity: 1;
  }
  white-space: nowrap;
  cursor: move;
  cursor: grab;
  cursor: -moz-grab;
  cursor: -webkit-grab;
`
```